cli/compose/types/types.go

```diff
@@ -4,6 +4,8 @@ import (
 	"encoding/json"
 	"fmt"
 	"time"
+
+	"github.com/docker/docker/api/types/mount"
```

Comment:
I think the reason these types were not imported was to make the compose-types not directly depend on docker API types, and to keep it "agnostic".

Now that the compose-spec exists, I don't think that should still be a concern, as the compose implementation in this repository is only for `docker stack`; users that want a more agnostic implementation are better off using the compose-spec as a library than the types from this repo.